src/Service/Search/SearchService/AbstractSearchHelper.php

```diff
@@ -46,7 +46,8 @@ public function __construct(
     public function addSearchRestrictions(
         SearchInterface $search,
         string $userPermission,
-        string $workspaceType
+        string $workspaceType,
+        PermissionTypes $permissionType = PermissionTypes::LIST
```

Comment:
I think we can remove some Default values? 
e.g this is used in search() which also has a default value..

Reply:
Hmm, I would agree if it were e.g. private methods. But those are all public and cann be called, so I would leave the default, unless we want to force the permissionType.

Reply:
Works for me :+1: